### ISSUE TITLE
simple 2 variable linear regresyon

### ISSUE THREAD

**Issue by alikara345** [unifyai/ivy#25744]
sklearn 

trk linear regresyon
![rttr](https://github.com/unifyai/ivy/assets/66207259/a3350046-b3d2-43d6-af74-7e50d15f7fea)

**Comment by alikara345** [comment]
`import ivy
import numpy as np
import pandas
import pandas as pd
import torch
from sklearn.model_selection import train_test_split
from sklearn.datasets import fetch_california_housing


def fllineerr(x):

    ivy.set.backend("Torc")

    data = pd.read_csv("data")
    X, y = data.data, data.target


    X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=0.7, shuffle=True)

    # Convert to 2D PyTorch tensors
    X_train = torch.tensor(X_train, dtype=torch.float32)
    y_train = torch.tensor(y_train, dtype=torch.float32).reshape(-1, 1)
    X_test = torch.tensor(X_test, dtype=torch.float32)
    y_test = torch.tensor(y_test, dtype=torch.float32).reshape(-1, 1)




`
